ignite/pkg/env/env.go

```diff
@@ -0,0 +1,11 @@
+package env
+
+import "os"
+
+const (
+	debug = "IGN_DEBUG"
```

Comment:
Can we just use `DEBUG`?

Reply:
Since envs are global, I'd rather not  take the risk of enabling one by mistake, that's why I think it's good to have a prefix. Any though on this @jeronimoalbi ?

Reply:
Agree, it is a good practice to use prefixes to namespace env variables to avoid unexpected behaviours 👍

Reply:
Ok makes sense